### PR TITLE
simplejson replaced by json in  4 test files

### PR DESCRIPTION
**Pull request by Enigma04**
Related to #133
The test files that were changed include:
* [ ] infogami/infobase/tests/test_client.py
* [ ] infogami/infobase/tests/test_save.py
* [ ] test/test_infobase.py
* [ ] tests/test_infogami/test_pages.py

### CODE REVIEW

File: infogami/infobase/tests/test_save.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import datetime
 
 import pytest
-import simplejson
+import json
```
Comment (cclauss) [inline infogami/infobase/tests/test_save.py:4]: Please also fix lines 82 and 98.

Comment (Enigma04) [reply infogami/infobase/tests/test_save.py:4]: done, you can check now

**Comment by cclauss** [comment]
#139 has put new automated testing in place.  Do you know how to git rebase to master?  If so please do so that tests are run on these changes.  If not then please let me know and I will perform a workaround.

**Comment by cclauss** [comment]
Please undo the changes to the two `infogami/utils` files and we can put those changes into a separate PR once this one is landed.

**Comment by Enigma04** [comment]
@cclauss for some reason I am unable to undo the commits of the utils files (I've tried hard and soft resets). So can I close this PR and create a new one just for the tests?

**Comment by Enigma04** [comment]
@cclauss I have reverted the changes done to the utils files

**Comment by Enigma04** [comment]
I have also rebased git to master from my IDE.

**Comment by cclauss** [comment]
How many of these four files are [tested by our GitHub Action](https://github.com/internetarchive/infogami/actions)?
How many of the files that are tested pass those tests?

**Comment by Enigma04** [comment]
@cclauss 
in `pytest infobase ` 25 passed and 70 errors
in `pytest tests` 30 passed, 7 skipped
in `pytest test` 12 skipped, 3 errors

but overall it is showing that all tests have been passed and there are no conflicts with the base branch

**Comment by cclauss** [comment]
The tests either all fail (Error) or are skipped (skip) so you will need to be patient until we can get the tests running properly (without `|| true`) on GitHub Actions before we can land this pull request.
* [infogami/infobase/tests/test_client.py EEEEEEEEEEEE](https://github.com/internetarchive/infogami/runs/1992451019#step:19:25)
* [infogami/infobase/tests/test_save.py EEEEEEEEEEEEEEEEEEEEEE](https://github.com/internetarchive/infogami/runs/1992451019#step:19:625)
* [test/test_infobase.py ssssssssss](https://github.com/internetarchive/infogami/runs/1992451019#step:21:13)
* [tests/test_infogami/test_pages.py ssssss](https://github.com/internetarchive/infogami/runs/1992451019#step:20:14)

**Comment by Enigma04** [comment]
@cclauss should I start fixing the other files?

**Comment by cclauss** [comment]
Yes but in a separate pull request please.

**Comment by cclauss** [comment]
Closing and reopening to rebase and retest...

**Review by cclauss — APPROVED** [review]
Now our GitHub Actions tests are all in place and these test files pass those tests.

Thanks for your contribution and thanks for your patience!!

**Pull request merged**